test/canaries/alerts/nrql_templates/agent_process_metrics.tftpl

```diff
@@ -0,0 +1,8 @@
+SELECT abs(
+filter(average(numeric(${metric})),WHERE displayName like '${display_name_current}')
+-
+filter(average(numeric(${metric})),WHERE displayName like '${display_name_previous}')
+)
+FROM ProcessSample
+WHERE commandLine like 'newrelic-infra'
```

Comment:
We should add the wildcard here ☝️ (probably already broken :( )
```
WHERE commandLine like '%newrelic-infra%'
```

Reply:
Good catch! Fixed https://github.com/newrelic/infrastructure-agent/pull/1804/commits/024d7fe813f736f08b0ba443b5ec3bbc180f10f2